packages/react/src/components/Card/CardMedia/Wrap.js

```diff
@@ -24,12 +24,11 @@ const mediaSizeStyles = {
 }
 
 export default styled('div')`
-  background: transparent no-repeat center center / cover;
+  background: #e1e8ed no-repeat center center / cover;
   display: block;
   overflow: hidden;
   height: auto;
   position: relative;
-  transition: flex-basis 0.25s ease-in-out;
```

Comment:
How come you removed this @Kikobeats?

Reply:
I didn't note it's necessary, that transition is only visible on viewport resize, right?

Reply:
Yeah correct, I don't usually add those things either tbh when it's just a resize transition. Let's bin it 👍